examples/workload/workloads_search_example.py

```diff
@@ -0,0 +1,156 @@
+                    '2': {'name': 'Search and Facet Compute Resources',
+                          'function_call': search_and_facet_compute_resources},
+                    '9': {'name': 'Quit',
+                          'function_call': quit_script}
+                    }
+
+    return menu_content
+
+
+def user_input(menu):
+    """Prompt user for menu choice"""
+    print('-' * 21)
+    for item in menu:
+        print(f'{item}. {menu[item]["name"]}')
+
+    choice = input('\nEnter your choice: ')
+    while choice not in str(menu.keys()):
+        choice = input('Enter a valid choice: ')
+
+    return choice
+
+
+def quit_script(cbc, menu, parser):
+    """Quits script"""
+    sys.exit()
+
+
+def fetch_compute_resource_by_id(cbc, menu, parser):
+    """Creates Fetch Compute Resource by ID SDK call"""
+    event_id = None
+    while not event_id:
+        event_id = input('\nPlease provide ID for the compute resource: ').strip()
+        try:
+            event_id = int(event_id)
+        except ValueError:
+            event_id = None
+            print('\nERR: ID must be integer!')
+
+    print(cbc.select(ComputeResource, event_id))
+    print('-' * 79)
+    new_query = input('\nMake anothery query? Y/n\n')
+    if new_query.lower() in ['y', 'yes']:
+        fetch_compute_resource_by_id(cbc, menu, parser)
+    else:
+        menu['0']['function_call']()
+
+
+def search_and_facet_compute_resources(cbc, menu, parser):
+    """Creates Search and Facet Compute Resources SDK call"""
+    print('\n' + '-' * 21)
+    print('Available criteria filters:\n')
+    print(*('appliance_uuid', 'eligibility', 'cluster_name', 'name',
+            'ip_address', 'installation_status', 'uuid', 'os_type',
+            'os_architecture'), sep='\n')
+    print('-' * 21)
+
+    query = cbc.select(ComputeResource)
+
+    use_filter = input('\nWould you like to use criteria filters?: Y/n\n')
+    if use_filter.lower() in ['y', 'yes']:
+        print('Each criteria accepts one or more space separated values.',
+              'Exmaple: --name ABCD --appliance_uuid 1234 5678')
+        args = parser.parse_args(input('Enter criteria filters: ').split())
+        if args.name:
+            query.set_name(args.name)
+        if args.os_type:
+            query.set_os_type(args.os_type)
+        if args.appliance_uuid:
+            query.set_appliance_uuid(args.appliance_uuid)
+        if args.cluster_name:
+            query.set_cluster_name(args.cluster_name)
+        if args.ip_address:
+            query.set_ip_address(args.ip_address)
+        if args.installation_status:
+            query.set_installation_status(args.installation_status)
+        if args.uuid:
+            query.set_uuid(args.uuid)
+        if args.os_architecture:
+            query.set_os_architecture(args.os_architecture)
+        if args.eligibility:
+            query.set_eligibility(args.eligibility)
+
+    print(*query)
```

Comment:
Does this execute the query or just print the query in string format? If it only prints the query then we should probably do loop here to print the compute resources that were found from the search

Reply:
It executes it. And the best part is that you can stack any number of filters.

Reply:
That's cool! Though I don't think you are correct with `the execution is done with the query.set_eligibility(args.eligibility)` as this area of the code does not perform the execution of the query https://github.com/carbonblack/carbon-black-cloud-sdk-python/blob/develop/src/cbc_sdk/workload/vm_workloads_search.py#L127

What I believe happens is the *query is likely treated as a iter-able function so the print statement loops through potentially.

Can you verify that this will work for more than one compute resource?

Reply:
https://user-images.githubusercontent.com/74309356/108233444-750a6200-714c-11eb-8219-3310b56188f6.mov

Reply:
Awesome! I like the use of the video for proof